Add `How advances in technology affected the osu! community` news post (#10659)
ppy/osu-wiki: news/2023/2023-12-22-how-advances-in-technology-affected-the-osu-community.md

---
layout: post

> title: How advances in technology affected the osu! community

Author: the first one makes the most sense to me, but the capitalization of "Community" really stings my eyes lol

> title: How advances in technology affected the osu! community

Reviewer: [discord dwellers were pretty meh about changing this](https://discord.com/channels/188630481301012481/1003689172420276246/1187493368494825614) so guess we're not

date: 2023-12-22 12:00:00 +0000
---

> Let's walk back through the past 15 years of osu!, and how various new technologies affected the game and the community.

Reviewer: ```suggestion
Let's take a stroll through the past 15 years of osu!, and see how various new technologies affected the game and the community.
```

![](/wiki/shared/news/2023-12-22-how-advances-in-technology-affected-the-osu-community/banner.jpg)

Every now and then there's a new piece of tech that's slated to change the way games are played, whether it be a new input device, a new way to squeeze more FPS out of your PC, or a whole new way to experience games.

> While the osu! community has managed to stay unswayed as innovations such as virtual reality, augmented reality, cloud gaming, and AI advancements cropped up due to them being inapplicable to osu!, there were still some that changed the way the game is played, or at least challenged the views on the most optimal way to play it.

Author: i think stuff like AI or ML mapping _probably is_ a thing but that's not really adopted or anything, I'd say its too early to mention it as a piece of tech that impacted us..... for now

> While the osu! community has managed to stay unswayed as innovations such as virtual reality, augmented reality, cloud gaming, and AI advancements cropped up due to them being inapplicable to osu!, there were still some that changed the way the game is played, or at least challenged the views on the most optimal way to play it.

Reviewer: also with art and whatnot

> While the osu! community has managed to stay unswayed as innovations such as virtual reality, augmented reality, cloud gaming, and AI advancements cropped up due to them being inapplicable to osu!, there were still some that changed the way the game is played, or at least challenged the views on the most optimal way to play it.

Author: you suggest I replace it then?

> While the osu! community has managed to stay unswayed as innovations such as virtual reality, augmented reality, cloud gaming, and AI advancements cropped up due to them being inapplicable to osu!, there were still some that changed the way the game is played, or at least challenged the views on the most optimal way to play it.

Author: https://github.com/ppy/osu-wiki/pull/10659/commits/fa9d7fc3fe8cd87e77436313080f5632d923a8e1 i just removed it altogether i really dont want people arguing over AI in the comments

The community somewhat sarcastically rallied around freedomdiver during one night in November, as they proceeded to break the pp record 3 times, all on the infamous [Yuima-ru*World TVver.](https://osu.ppy.sh/beatmapsets/459149#osu/994495) map. Hundreds of players spectated freedomdiver as they broke the pp record, and thanks to [CPUGeek](https://osu.ppy.sh/users/4172839) you can experience a snippet of the atmosphere surrounding the play:

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/JCU2ZWRj5gg" frameborder="0" allowfullscreen></iframe>
</div>

A month later, on December 15th, touch screen scores had their pp reduced along with an introduction of the brand new "Touch Device" mod, which would reduce the amount of pp gained from jumps on any map when osu! detected touch screen being used as a gameplay device.

> Nowadays, a new, unique usage for touch screen was figured. Some players use it exclusively as an aiming device, while still tapping to the beat with their good ol' keyboards. You can look at [bonk](https://osu.ppy.sh/users/4209965) and their [first person live play](https://www.youtube.com/watch?v=PE91EAzRcUo) to see what it looks like.

Reviewer: there's no way this is a recent discovery. iirc some old cintiq players played this way

```suggestion
In the end, tapping and aiming with two hands isn't the only way to use a touch screen. Some players use them exclusively as aiming devices, while still tapping to the beat with their good ol' keyboards. You can look at [bonk](https://osu.ppy.sh/users/4209965) and their [first-person live play](https://www.youtube.com/watch?v=PE91EAzRcUo) to see what it looks like.
```

Author: yeah probably isnt new

> ## "Rapid trigger" feature on mechanical keyboards (2021)

Reviewer: ```suggestion
## Rapid trigger (2021)
```

> "[Rapid trigger](https://wooting.io/rapid-trigger)" is a new keyboard software feature first implemented by Wooting, a mechanical keyboard manufacturer. Its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect) allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether. The animation below visually represents how it works:

Reviewer: ```suggestion
"[Rapid trigger](https://wooting.io/rapid-trigger)" is a new keyboard software feature first implemented by Wooting, a mechanical keyboard manufacturer. Its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect) allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether. The animation below (by Wooting) visually represents how it works:
```

> "Rapid trigger" is a new keyboard software feature that's first been implemented by Wooting, a mechanical keyboard manufacturer. Due to its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect), it allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether by using rapid trigger. The animation below visually represents how it works:

Reviewer: ```suggestion
"Rapid trigger" is a new keyboard software feature first implemented by Wooting, a mechanical keyboard manufacturer. Its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect) allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether. The animation below visually represents how it works:
```

![](/wiki/shared/news/2023-12-22-how-advances-in-technology-affected-the-osu-community/rapid_trigger_example.gif)

> This has caused quite the turmoil amongst the community, as some believe this greatly reduces the skill required to be considered "good at speed", and lowers the skill ceiling quite significantly by allowing players to be less precise with their technique, but still make inputs. The other side of the community argues that this is only beneficial for players who "mash" keys instead of trying to accurately hit notes.

Reviewer: ```suggestion
This has caused quite the turmoil amongst the community, as some believe this greatly reduces the skill required to be considered "good at speed", and lowers the skill ceiling quite significantly by allowing players to be less precise with their technique, but still not miss inputs or break combo as often. The other side of the community argues that this is only beneficial for players who "mash" keys instead of trying to accurately hit notes.
```

The arguments are still a topic of countless heated debates, but if you want to get an opinion of a pro player, look no further than [BTMC](https://osu.ppy.sh/users/3171691)'s video on the rapid trigger feature:

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/cFvdM_0XuAg" frameborder="0" allowfullscreen></iframe>
</div>

---

> This is a fascinating way to look back through the ages and remind ourselves of a different, simpler time. When the community did not have to worry about the best keyboards and tablets to use to become the best player instantaneously, and just enjoyed osu! for what it is, a rhythm game with an amazing community and one of the largest selections of beatmaps/charts in any rhythm game ever.

Reviewer: that comparison may not be the best one to make

```suggestion
This is a fascinating way to look back through the ages and remind ourselves of a different, simpler time, when the community did not have to worry about the best keyboards and tablets to use to become the best player instantaneously, and just enjoyed osu! for what it is — a rhythm game with an amazing community creating and playing awesome beatmaps.
```
